src/auth/session/session-utils.ts

```diff
@@ -0,0 +1,44 @@
+import type {SessionInterface} from './types';
```

Comment:
This is old code just moved to a separate file so that the linter stops complaining.